interfaces/Portalicious/src/app/domains/registration/registration.helper.ts

```diff
@@ -53,6 +54,18 @@ export const REGISTRATION_STATUS_VERB: Record<RegistrationStatusEnum, string> =
     [RegistrationStatusEnum.deleted]: $localize`Delete`,
   };
 
+export const LANGUAGE_ISO_639_1: Record<LanguageEnum, string> = {
```

Comment:
I don't think this should live in this file. Maybe `~/utils/locale`?

Reply:
I see. If that is solely how `LanguageEnum` is used, then maybe a clearer name for it would be `PreferredLanguageEnum` - but maybe out of scope 😄 

For now this makes sense to me.